README.md

```diff
@@ -345,5 +345,5 @@ Users can choose between testing Jenkins based on a RHEL or CentOS image.
     ```
 
 **Notice: By omitting the `VERSION` parameter, the build/test action will be performed
-on all provided versions of Jenkins. Since we are currently providing only version `1`,
+on all provided versions of Jenkins. Since we are currently providing only version `2`,
```

Comment:
This is the only non-whitespace change in this file. I hadn't realised that my editor had removed trailing spaces on other lines. If you'd prefer, I can modify the commit to re-add those trailing spaces. Otherwise you can get this page to ignore whitespace-only chunks, by adding `?w=1` to the URL: https://github.com/openshift/jenkins/pull/573/files?w=1